captum/module/binary_concrete_stochastic_gates.py

```diff
@@ -147,7 +146,35 @@ def forward(self, *args, **kwargs):
                 model loss,
                 e.g. loss(model_out, target) + l0_reg
         """
-        return super().forward(*args, **kwargs)
+        return super().forward(input_tensor)
+
+    def get_gate_values(self, clamp: bool = True) -> Tensor:
+        """
+        Get the gate values, which are the means of the underneath gate distributions,
+        optionally clamped within 0 and 1.
+
+        Returns:
+            gate_values (Tensor): value of each gate in shape(n_gates)
+
+            clamp (bool): if clamp the gate values. As smoothed Bernoulli
```

Comment:
nit: maybe `indicates whether to clamp the gate values or not` ? (also in gaussian stochastic gates)